src/trex.ts

```diff
@@ -0,0 +1,190 @@
+const dependenciesGenerator: Fig.Generator = {
+  script: "cat import_map.json",
+  postProcess: function (out) {
+    if (out && out !== "") {
```

Comment:
This check is enough, in JS an empty string returns false when boolean tested:
```suggestion
    if (out) {
```

Reply:
Of course, silly me!